day3/slice_timing_solutions.ipynb

```diff
@@ -646,7 +646,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython2",
-   "version": "2.7.9"
+   "version": "2.7.10"
```

Comment:
You might have committed all the files with something like ``git commit -a``.  I highly recommend you add all your files for the commit one by one with ``git commit add my_file.txt`` - that way it's much harder to accidentally add things you don't really want - see : https://matthew-brett.github.io/pydagogue//curious_git.html#adding-with-a